src/PluginDefinition.cpp

```diff
@@ -296,6 +296,12 @@ void onBeforeSave(HWND hWnd)
         }
     }
 
+    // Save the folding behavior and set it to 0 to keep folds from opening
+    // when modifying the file.
+    HWND curScintilla = getCurrentScintilla();
+    int automatic_fold = SendMessage(curScintilla, SCI_GETAUTOMATICFOLD, 0, 0);
+    SendMessage(curScintilla, SCI_SETAUTOMATICFOLD, 0, 0);

```

Comment:
Is this line necessary? It seems that you will send the automatic fold flag any way.

Reply:
Yes. The automatic fold must be set to 0 (disabled) before modifying the file.

Reply:
Thanks!